docs/content/docs/connectors/datastream/overview.md

```diff
@@ -44,7 +44,7 @@ Connectors provide code for interfacing with various third-party systems. Curren
  * [Elasticsearch]({{< ref "docs/connectors/datastream/elasticsearch" >}}) (sink)
  * [FileSystem (Hadoop included) - Streaming only sink]({{< ref "docs/connectors/datastream/streamfile_sink" >}}) (sink)
  * [FileSystem (Hadoop included) - Streaming and Batch sink]({{< ref "docs/connectors/datastream/file_sink" >}}) (sink)
- * [FileSystem (Hadoop included) - Batch source] ({{< ref "docs/connectors/datastream/formats" >}}) (source)
+ * [FileSystem (Hadoop included) - Batch source]({{< ref "docs/connectors/datastream/formats" >}}) (source)
```

Comment:
This is another typo.

Reply:
That's a missing space. That's not worth another commit.

Reply:
https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/connectors/datastream/overview/

[FileSystem (Hadoop included) - Batch source] (//nightlies.apache.org/flink/flink-docs-release-1.14/docs/connectors/datastream/formats/) (source)